rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -179,8 +181,29 @@ Apply Configuration
           - http://flyteadmin.flyte.svc.cluster.local:80
 
    Save and exit your editor.
+   
+#. Replace the default `clientSecret` for `flytepropeller` using an encoded password of your choice:
 
-#. Restart `flyteadmin` for the changes to take effect:
+   .. prompt:: bash
```

Comment:
If I remember correctly, the non-hashed version of the password had to be entered at another location in the helm values file, right?

Reply:
Yes, it should go under:

```yaml
configmap:
  adminServer:
    auth:
        selfAuthServer:
          staticClients:
            flytepropeller:
              id: flytepropeller
              client_secret: <secret>
              redirect_uris:
                - "http://localhost:3846/callback"
              grant_types:
                - refresh_token
                - client_credentials
              response_types:
                - token
              scopes:
                - all
                - offline
                - access_token

```

Reply:
I don't think this is enough though - i think you need all three because it won't merge, it'll replace the whole array

```appAuth:
  thirdPartyConfig:
    flyteClient:
      clientId: flytectl
      redirectUri: http://localhost:53593/callback
      scopes:
      - offline
      - all
  selfAuthServer:
    staticClients:
      flyte-cli: 
        id: "flyte-cli"
        redirect_uris:
          - "http://localhost:53593/callback"
          - "http://localhost:12345/callback"
        grant_types:
          - refresh_token
          - authorization_code
        response_types:
          - code
          - token
        scopes:
          - all
          - offline
          - access_token 
        public: true
      flytectl: 
        id: flytectl
        redirect_uris:
          - "http://localhost:53593/callback"
          - "http://localhost:12345/callback"
        grant_types:
          - refresh_token
          - authorization_code
        response_types:
          - code
          - token
        scopes:
          - all
          - offline
          - access_token 
        public: true
      flytepropeller: 
        id: flytepropeller
        client_secret: JDJhJDAyourhashedsecretAKAoutputofthepythononeliner
        redirect_uris:
          - "http://localhost:3846/callback"
        grant_types:
          - refresh_token
          - client_credentials
        response_types:
          - token
        scopes:
          - all
          - offline
          - access_token 
```

Reply:
Should also mention that this entire `selfAuthServer` section should be deleted if you don't want to use Flyte's internal authorization server.  If you're using Okta for example as your IdP, you can use Okta's auth server, and create these three applications there.  Google oidc doesn't have this though, which is why we built one into Flyte.

Reply:
WIth "all three" you mean `flytecli`, `flytectl` and `flypropeller` as `staticClients`? 
As far as locations where to update the `flytepropeller` secret I can count 2 so far

Reply:
yeah sorry, the three clients.  since it's a list they'll get lost in the merge.

Reply:
Thank you @fg91 and @andrusha. I hope the latest commit reflects better your findings.
@wild-endeavor I chose not to mention the possibility of removing the `selfAuthServer` as there is a separate section in the docs for the external auth server, where we could mention this. Also I confirmed the `values.yaml` has the three clients there